app/helpers/form_errors_builder.rb

```diff
@@ -0,0 +1,29 @@
+class FormErrorsBuilder < ActionView::Helpers::FormBuilder
```

Comment:
I take the criticism that the method has been poorly implemented in terms of putting html in there when that is not its responsibility. I think passing this on to a view when we have encapsulated every thing else about the form into a form helper is just wrong, but I don't see us reaching agreement on that side of things.